### PR TITLE
Prop table permalink

### PR DESCRIPTION
**Pull request by klauspaiva**
Allows permalinks to be rendered as each prop displays.

Loom demo: https://www.loom.com/share/85edd8c9a0a2435dba5cff7989651ed0

The demo above was recorded before the correct colour was given by design, so this is a more accurate screenshot of what the final experience can look like:

Screen Shot 2022-05-12 at 1 45 04 pm

Of course, when `LayoutRenderer` is used it's entirely up to the developer to handle styles to whatever visual they want to achieve.

### CODE REVIEW

**Review by lorvsso — APPROVED** [review]
Looks great to me, love this change. @danieldelcore is a lot more familiar with this than I am so his review is probably worth waiting for but it's a tick from me!

File: packages/babel-plugin-extract-react-types/index.js [hunk 1]
```diff
@@ -28,6 +28,15 @@ module.exports = babel => {
                     )
                   )
                 );
+                programPath.node.body.push(
+                  t.expressionStatement(
+                    t.assignmentExpression(
+                      '=',
+                      t.memberExpression(t.identifier(name), t.identifier('___displayName')),
+                      t.stringLiteral(name)
```
Comment (danieldelcore) [inline packages/babel-plugin-extract-react-types/index.js:36]: YESSSS

**Review by danieldelcore — APPROVED** [review]
LGTM

**Comment by danieldelcore** [comment]
Thanks for taking the long way around @klauspaiva! This is a great addition !

**Pull request merged**